### DISCUSSION TITLE
Experiencing this bug when I try to push changes from my local machine to github just after I added image from GIT Bash .

### DISCUSSION THREAD

**Discussion by shivambhat** [EddieHubCommunity/support#301] [Q&A] [answered]
**ERROR:**
![error](https://user-images.githubusercontent.com/18063625/95087610-c272bf00-06d6-11eb-8e2a-e920ddda0cb1.jpg)

shivbhat@HY16-5CG7284RYL MINGW64 /C/Users/shivbhat/Desktop/learngitrepo/Hacktoberfest2020/static/images (master)
$ git push origin master
remote: Permission to OpenSouceCode/Hacktoberfest2020.git denied to shivambhat.
fatal: unable to access 'https://github.com/OpenSouceCode/Hacktoberfest2020/': The requested URL returned error: 403

**Comment by shivambhat** [comment]
Error

**Comment by eddiejaoude** [comment]
It says you do not have permission to push to the repo. Is that repo yours?

**Reply by shivambhat** [reply]
No its a public repo

**Reply by shivambhat** [reply]
I have  already cloned this rep so that I can access it through my  local machine and I tried pushing changes to this repo ,but I  keep getting this error again and again & I can assure you that my credentials are correct since they are the same which i used to  Push and pull changes on my own  repo on Github

**Comment by mikeysan** [comment, marked as answer]
@shivambhat Did you create a new branch to push to?
These are the instructions from the repo
```

    Fork this Repository using the button at the top
    Clone your forked repository to your pc ( git clone "url from clone option.)
    Create a new branch for your modifications (ie. git branch new-user and check it out git checkout new-user and git checkout -b new-user)
    Add your profile image in static/images/ ( use drag and drop option or upload by commands.)
    Add your profile data in content/participant/
    Add your files (git add -A), commit (git commit -m "added myself") and push (git push origin new-user)
    Create a pull request
    Star this repository
```
Might the issue be that you are attempting to push to 'master'?

**Reply by shivambhat** [reply]
Thanks its working now, I guess I was  cloning the wrong  forked repo.

Its sorted now ,anyways Thanks a lot 👍
